ansible/roles/debops.phpipam/defaults/main.yml

```diff
@@ -10,240 +10,365 @@
+# .. envvar:: phpipam__scripts_config_output [[[
 #
 # Default configuration file location
-phpipam_scripts_config_output: '/etc/dhcp/dhcpd-hosts.conf'
+phpipam__scripts_config_output: '/etc/dhcp/dhcpd-hosts.conf'
 
                                                                    # ]]]
-# .. envvar:: phpipam_scripts_config_restart_command [[[
+# .. envvar:: phpipam__scripts_config_restart_command [[[
 #
 # A command to issue after the configuration file is generated and changes were
 # noticed.
-phpipam_scripts_config_restart_command: '/etc/init.d/isc-dhcp-server restart'
+phpipam__scripts_config_restart_command: '/etc/init.d/isc-dhcp-server restart'
 
                                                                    # ]]]
-# .. envvar:: phpipam_scripts_config_groups [[[
+# .. envvar:: phpipam__scripts_config_groups [[[
 #
 # Default configuration for phpipam-hosts script. It will create
 # a ``/etc/dhcp/dhcpd-hosts.conf`` file with all hosts configured in phpIPAM,
 # or just the sections/subnets specified above. If any changes are detected,
 # DHCP server (by default ISC DHCP) will be restarted.
-phpipam_scripts_config_groups:
+phpipam__scripts_config_groups:
   'hosts':
-    sections: '{{ phpipam_scripts_config_sections }}'
-    subnets: '{{ phpipam_scripts_config_subnets }}'
+    sections: '{{ phpipam__scripts_config_sections }}'
+    subnets: '{{ phpipam__scripts_config_subnets }}'
     active: 'true'
     reserved: 'false'
     offline: 'false'
     dhcp: 'false'
-    output: '{{ phpipam_scripts_config_output }}'
-    restart_command: '{{ phpipam_scripts_config_restart_command }}'
+    output: '{{ phpipam__scripts_config_output }}'
+    restart_command: '{{ phpipam__scripts_config_restart_command }}'
     restart: 'true'
 
                                                                    # ]]]
-# .. envvar:: phpipam_scripts_cron_period [[[
+# .. envvar:: phpipam__scripts_cron_period [[[
 #
 # phpipam-hosts regeneration period in minutes (cron format)
-phpipam_scripts_cron_period: '*/5'
+phpipam__scripts_cron_period: '*/5'
+                                                                   # ]]]
+# ]]]
+
+# Configuration for other Ansible roles [[[
+# -----------------------------------------
 
+# .. envvar:: phpipam__php__dependent_packages [[[
+#
+# List of PHP packages to install using :ref:`debops.php` role.
+phpipam__php__dependent_packages:
+  - 'mysql'
+  - 'gmp'
+  - 'gd'
+  - 'curl'
+  - 'mcrypt'
+  - 'ldap'
+  - 'php-pear'
+                                                                   # ]]]
+# .. envvar:: phpipam__php__dependent_pools [[[
+#
+# PHP-FMP pool configuration for the  :ref:`debops.php` Ansible role.
+phpipam__php__dependent_pools:
+
+  - name: 'phpipam'
+    enabled: True
+    user: '{{ phpipam__user }}'
+    group: '{{ phpipam__group }}'
+                                                                   # ]]]
+
+
+# .. envvar:: phpipam__nginx__dependent_upstreams [[[
```

Comment:
Remove the empty lines between the variables for better vim folding.

Reply:
Done